app/dts/bindings/zmk,soft-off-wakeup-sources.yaml

```
@@ -0,0 +1,14 @@
+# Copyright (c) 2023 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+description: |
+  Description of all possible wakeup-sources from a forces
```

```suggestion
  Description of all possible wakeup-sources from a
```
Extra word? Or was that supposed to be "forced"?